src/Pages/Cart/Cart.js

```diff
@@ -154,7 +153,7 @@ function Cart() {
             ) : (
                 <></>
             )}
-            <div className="CartPage">
+            <div className={payment ? "CartPage noscroll" : "CartPage"}>
```

Comment:
suggest:
isPayment.
for booleans, you can use is before actual variables.
ex:
isPayment,
isModalOpen
isEditable